package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "razzle start",
     "build": "razzle build",
     "test": "npm run test:lint && npm run test:unit",
-    "test:lint": "eslint --ext .js,jsx ./src",
+    "test:lint": "eslint --ext .js,jsx ./src && eslint --ext .json ./data",
```

Comment:
Don't need to run the command twice - just add `.json` to `--ext`, and append `./data` after `./src`

Reply:
@jtart you're right thanks, change made